Find a functionally equivalent code:android.print.PrintManager.getInstalledPrintServices:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Gets the list of installed print services.
 *
 * @return The installed service list or an empty list.
 * @hide
 */

@@ -1,4 +1,8 @@
 {
+    if (mService == null) {
+        Log.w(LOG_TAG, "Feature android.software.print not available");
+        return Collections.emptyList();
+    }
     try {
         List<PrintServiceInfo> installedServices = mService.getInstalledPrintServices(mUserId);
         if (installedServices != null) {

